config/eslint/.eslintrc.js

```diff
@@ -1,105 +1,96 @@
-const path = require("path");
+/**
+ * eslint config
+ *
+ * @licence REWE Digital GmbH
+ */
+
 module.exports = {
-    "extends": [
-        "eslint:recommended",
-        "standard",
-        "eslint-config-crockford",
-        "plugin:import/errors",
-        "plugin:import/warnings"
+    'extends': [
+        'eslint:recommended',
+        'standard'
     ],
-    "plugins": [
-        "m99coder",
-        "standard",
-        "import"
+    'plugins': [
+        'standard'
     ],
-    "parserOptions": {
-        "ecmaVersion": 6,
-        "sourceType": "module",
-        "ecmaFeatures": {
-            "jsx": false,
-            "impliedStrict": true
-        },
-        "useJSXTextNode": true
+    'parserOptions': {
+        'ecmaVersion': 5,
+        'sourceType': 'script'
     },
-    "root": true,
-    "env": {
-        "browser": true,
-        "jquery": true,
-        "es6": true,
-        "jest": true,
-        "node": true,
-        "builtin": true
+    'root': true,
+    'env': {
+        'browser': true,
+        'jquery': true
     },
-    "globals": {
-        "$": false,
-        "jQuery": true,
-        "console": true,
-        "module": true,
-        "window": true,
-        "document": true,
-        "require": true,
-        "_": true
+    'globals': {
+        '$': false,
+        'jQuery': true,
+        'console': true,
+        'module': true,
+        'window': true,
+        'document': true,
+        'require': true,
+        '_': true,
+        'pageData': true, // tracking object for CMP
+        '_satellite': true // Adobe tracking function
     },
-    "rules": {
-        "new-cap": 2,
-        "no-caller": 2,
-        "no-eq-null": 2,
-        "indent": [
-            "error",
+    /**
+     * 0 = turned off
+     * 1 = warning
+     * 2 = error
+     */
+    'rules': {
+        'new-cap': 1,
+        'no-caller': 0,
```

Comment:
suggesting at least 1 here - ecmascript 5 in strict mode disallows the use of caller anyways